src/decoder_util.c

```diff
@@ -51,7 +51,7 @@ static char *bitrow_asprint_code(uint8_t const *bitrow, unsigned bit_len)
     row_bytes[0] = '\0';
     // print byte-wide
     for (unsigned col = 0; col < (unsigned)(bit_len + 7) / 8; ++col) {
-        sprintf(&row_bytes[2 * col], "%02x", bitrow[col]);
+        sprintf(&row_bytes[2 * col], "%02x", bitrow[col]); // NOLINT
```

Comment:
don't nolint `sprintf()`, we should rather fix those. I already got the easy ones 8f6032a

Reply:
Ok, I'll remove the annotations.

I've made some easy fixes, but I wonder what you prefer for the next case: change the length of the variable to 3 and use sizeof, leave it as size 4 and use fixed value 3 or sizeof?

https://github.com/merbanan/rtl_433/pull/2683/files#diff-3deacdaee1d97c163db52aa550e5802735d32fd07170be2408627c2aadc585e6R104

Reply:
The buffer should be as close as possible (so we can easily check) and the size wouldn't matter, but 3 here.
```
    // Extract the id as hex string
    char id[3];
    snprintf(id, sizeof(id), "%02X", b[1]); // FIXME: id should be a plain int really
```

Reply:
Should I remove the NOLINT on strcpy as well?

```c
        strcpy(topic, data->value.v_ptr); // NOLINT
```